app/config/staging.js

```diff
@@ -15,5 +15,13 @@ module.exports = {
     secretKey: 'miniostorageengine',
     bucket: 'rra',
     region: 'us-east-1'
+  },
+  analysisProcess: {
```

Comment:
I usually add them in so there's a base to copy for local.
Should we add them in with a comment in front, or remove the others as well?